### PR TITLE
Update RELEASE.md and README.md for release note.

### PR DESCRIPTION
**Pull request by yongtang**
Signed-off-by: Yong Tang <yong.tang.github@outlook.com>

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -1,3 +1,24 @@
...
+* Add ArrowIOTensor with input from a pyarrow.Table.
+* Add Numpy dataset support for numpy file and memory input.
+* Add decode_avro/encode_avro for Avro serialization support.
+* Prometheus dataset now outputs structued shapes.
+
+## Thanks to our Contributors
+
+This release contains contributions from many people:
+
+Bryan Cutler, DylanTallchiefGit, Kota Yamaguchi, Mark Daoust,
```
Comment (terrytangyuan) [inline RELEASE.md]: Use `@DylanTallchiefGit` for users without a name?

File: RELEASE.md [hunk 2]
```diff
@@ -1,3 +1,24 @@
+# Release 0.12.0
+
+## Major Features
+* Add Windows support for tensorflow-io.
+* Add PostgreSQL server dataset support.
+* Add MP3 format for audio dataset support.
+* Add ArrowIOTensor with input from a pyarrow.Table.
+* Add Numpy dataset support for numpy file and memory input.
+* Add decode_avro/encode_avro for Avro serialization support.
+* Prometheus dataset now outputs structued shapes.
```
Comment (terrytangyuan) [inline RELEASE.md]:
```suggestion
* Prometheus dataset now outputs structured shapes.
```

Comment (yongtang) [reply RELEASE.md]: @terrytangyuan Updated.

File: RELEASE.md (hunk 1 above)
Comment (yongtang) [reply RELEASE.md]: @terrytangyuan We respect contributor's preference on GitHub handle/name. We can always update later if contributor decide to make a change in handle/name.

Comment (terrytangyuan) [reply RELEASE.md]: Sounds good

**Pull request merged**